### ISSUE TITLE
Validator: Improve `!important` CSS marker error messages

### ISSUE THREAD

**Issue by Gregable** [ampproject/amphtml#27505]
Currently, the validator emits the following error messages for `!important` qualifiers in CSS:

In `<style amp-custom>` tags:
```
The text inside tag 'style amp-custom' contains 'CSS !important' which is disallowed.
```

In `style` attributes:
```
The attribute `style` in tag `X` is set to the invalid value `CSS !important`.
```

These are reused errors for more generic issues.

We should create a new validator error type and string for both of these, to make the error more clear. I propose:

```
Usage of the !important CSS qualifier is not allowed.
```
Which will link to https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml#stylesheets

**Comment by honeybadgerdontcare** [comment]
I think https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/style_pages/ may be a better link for the error message since `!important` is listed at the beginning.

**Comment by honeybadgerdontcare** [comment]
Fixed in cl/325253809. Will be merged to GitHub in the next validator roll-up
